### PR TITLE
Fix for entrypoint handling

### PR DESCRIPTION
**Pull request by natacha-beck**
Should fix the following #542

### CODE REVIEW

File: tools/python/boutiques/importer.py [hunk 1]
```diff
@@ -99,9 +99,30 @@ def get_entry_point(self, input_descriptor):
...
-                entrypoint = split[1].strip("[]\"")
+            if line.startswith("ENTRYPOINT"):
+                entrypoint_values = line.strip("ENTRYPOINT").strip()
+
+                # According to DOCKER documetation the ENTRYPOINT can
+                # be like:
+                #   > ENTRYPOINT ["executable", "param1", "param2"]
+                # or:
+                #   > ENTRYPOINT command param1 param2
+                if not entrypoint_values:
```
Comment (gkiar) [inline tools/python/boutiques/importer.py]: Thanks, @natacha-beck ! Just in the interest of perhaps making it a bit more readable and Pythonic, we might be able to simplify the logic to the following. The suggestion would replace everything up to line 126. What do you think?

```suggestion
                if not entrypoint_values:
                    return None

                if entrypoint_values.startswith("["):
                    # Replacing entrypoint_values with the Python-interpreted list version
                    entrypoint_values = json.loads(entrypoint_values)
                    # Adding single quotes around list items with spaces therein
                    entrypoint_values = [ev
                                         if " " not in ev else
                                         "'{0}'".format(ev)
                                         for ev in entrypoint_values]
                    return " ".join(entrypoint_values)

                return entrypoint_values
```

File: tools/python/boutiques/importer.py [hunk 2]
```diff
@@ -99,9 +99,43 @@ def get_entry_point(self, input_descriptor):
...
+                    entrypoint_values = json.loads(entrypoint_values)
+                    # Adding single quotes around list items with spaces therein
+                    entrypoint_values = [ev
+                                         if " " not in ev else
+                                         "'{0}'".format(ev)
+                                         for ev in entrypoint_values]
+                    return " ".join(entrypoint_values)
+
+                return entrypoint_values
+                    entrypoint = None
```
Comment (gkiar) [inline tools/python/boutiques/importer.py]: Hi @natacha-beck - this and the following lines in this block now need to be deleted. Thanks!

Comment (natacha-beck) [reply tools/python/boutiques/importer.py]: @gkiar should be ok now.

**Comment by glatard** [comment]
Hi @natacha-beck, thanks for working on this! The tests are currently failing due to pycodestyle (see Travis):
```
boutiques/importer.py:114:81: E501 line too long (90 > 80 characters)

The command "pycodestyle --max-line-length=80 boutiques/*.py boutiques/tests/*.py" exited with 1.
```

**Comment by glatard** [comment]
Hi @natacha-beck, the tests are now failing following the release of CentOS8 last week. We fixed that issue with @DarrinFong yesterday, would you like to pull `develop` in your branch so we can check that everything now passes?

**Comment by glatard** [comment]
Merging, as the test failures should be fixed in develop.

**Pull request merged**